cmd/segment-reaper/delete.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"encoding/csv"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/process"
+	"storj.io/storj/satellite/metainfo"
+)
+
+var (
+	deleteCmd = &cobra.Command{
+		Use:   "delete input_file.csv [flags]",
+		Short: "Deletes zombie segments from DB",
+		Args: func(cmd *cobra.Command, args []string) error {
+			if err := cobra.ExactArgs(1)(cmd, args); err != nil {
+				return err
+			}
+			if err := cobra.OnlyValidArgs(cmd, args); err != nil {
```

Comment:
I'm not understanding what this does without having `ValidArgs` field set.

On the other hand, I tried to set it to see in what it results and I don't think that we needed because it's for comparing the values of the positional parameters with static values.

Reply:
I think I misunderstood `OnlyValidArgs` because I copied it from different command. I will use only `ExactArgs`.